### PR TITLE
Local dev instructions added

### PR DESCRIPTION
**Pull request by har00ga**
### Description

**Write why you are making the changes in this pull request**
As requested

**Write a summary of the changes you are making**
Interop instructions have been merged to the main README in addition to a number of formatting adjustments

### CODE REVIEW

**Comment by rauljordan** [comment]
Hi @har00ga thanks for this! Sorry I wasn't clear enough on the instructions, but the text mostly looks good. What we're looking for is instructions for users to setup a local chain, not necessarily interop experts. So maybe we can rephrase the section to `Setting Up a Local Eth2 Chain` or something of that sort? We want to cater to those users that come to our discord for the first time, aren't as familiar with deep eth2 internals, and just want to set things up on their computer for testing and experimenting. 

The instructions themselves are nice, it's just a matter of phrasing it better for beginners. I'd also suggest removing the `launching from genesis.ssz` section and just including the quickstart chain option (the one with the interop flags).

**Comment by har00ga** [comment]
Ah understood, not a problem. :) Sorry for the miscommunication on the scope! I'll make updates this afternoon.

File: README.md [hunk 1]
```diff
@@ -1,28 +1,36 @@
...
+A more detailed set of installation and usage instructions as well as breakdowns of each individual component are available in the [official documentation portal](https://prysmaticlabs.gitbook.io/prysm/). If you still have questions, feel free to stop by either our [Discord](https://discord.gg/KSA7rPr) or [Gitter](https://gitter.im/prysmaticlabs/geth-sharding?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) and a member of the team or our community will be happy to assist you.
 
 **Interested in what's next?** Be sure to read our [Roadmap Reference Implementation](https://github.com/prysmaticlabs/prysm/blob/master/docs/ROADMAP.md) document. This page outlines the basics of sharding as well as the various short-term milestones that we hope to achieve over the coming year.
```
Comment (rauljordan) [inline README.md]:
```suggestion
```

This stuff was super outdated, rather just remove it entirely

File: README.md [hunk 2]
```diff
@@ -168,15 +176,17 @@ bazel run //beacon-chain -- --datadir=/tmp/prysm-data
...
 
-## Staking ETH: Running a Validator Client
+## Staking ETH: Running a validator client
 
 Once your beacon node is up, the chain will be waiting for you to deposit 3.2 Goerli ETH into the Validator Deposit Contract to activate your validator (discussed in the section below). First though, you will need to create a validator client to connect to this node in order to stake and participate. Each validator represents 3.2 Goerli ETH being staked in the system, and it is possible to spin up as many as you desire in order to have more stake in the network.
 
-### Activating Your Validator: Depositing 3.2 Goerli ETH
+For more information on the functionality of validator clients, see [this section](https://prysmaticlabs.gitbook.io/prysm/how-prysm-works/validator-clients) of our official documentation.
+
+### Activating your validator: Depositing 3.2 Goerli ETH
```
Comment (rauljordan) [inline README.md]:
```suggestion
### Activating your validator: depositing 3.2 Goerli ETH
```

File: README.md [hunk 3]
```diff
@@ -187,20 +197,59 @@ bazel run //validator
 ```
 **Congratulations, you are now running Ethereum 2.0 Phase 0!**
 
+## Setting up a local ETH2 development chain
+
+This section outlines the process of setting up Prysm for local interop testing with other Ethereum 2.0 client implementations. See the [INTEROP.md](https://github.com/prysmaticlabs/prysm/blob/master/INTEROP.md) file for advanced configuration options. For more background information on interoperability development, see [this blog post](https://blog.ethereum.org/2019/09/19/eth2-interop-in-review/).
+
+### Installation and Dependencies
```
Comment (rauljordan) [inline README.md]:
```suggestion
### Installation and dependencies
```

File: README.md [hunk 4]
```diff
@@ -187,20 +197,59 @@ bazel run //validator
...
+
+To begin setting up a local ETH2 development chain, follow the **Bazel** instructions found in the [dependencies](#dependencies) and [installation](#installation) sections respectively. After completion, continue on to [generating a genesis state](#generating-a-genesis-state).
```
Comment (rauljordan) [inline README.md]:
```suggestion
To begin setting up a local ETH2 development chain, follow the **Bazel** instructions found in the [dependencies](#dependencies) and [installation](#installation) sections respectively.
```

File: README.md [hunk 5]
```diff
@@ -1,28 +1,36 @@
...
 
 - [Dependencies](#dependencies)
 - [Installation](#installation)
-    - [Build Via Docker](#build-via-docker)
-    - [Build Via Bazel](#build-via-bazel)
-- [Running an Ethereum 2.0 Beacon Node](#running-an-ethereum-20-beacon-node)
-- [Staking ETH: Running a Validator Client](#staking-eth-running-a-validator-client)
+    - [Build via Docker](#build-via-docker)
+    - [Build via Bazel](#build-via-bazel)
+- [Running a beacon node](#running-a-beacon-node)
```
Comment (rauljordan) [inline README.md]: Can we do something instead like

```
- [Connecting to the Public Testnet]
    - [Running a Beacon Node]
        ...
    - [Staking ETH: Running a Validator Client]
        ...
- [Setting Up a Local Development Chain]
    ...

File: README.md [hunk 6]
```diff
@@ -88,12 +96,13 @@ Target //beacon-chain:beacon-chain up-to-date:
 
 In the example above, the beacon chain binary has been created in `bazel-bin/beacon-chain/linux_amd64_stripped/beacon-chain`.
 
-## Running an Ethereum 2.0 Beacon Node
+## Running a beacon node
```
Comment (rauljordan) [inline README.md]: +1 on the comment above, create a section called `Connecting to the Public Testnet` with brief mentions about how these are instructions to run a beacon+validator connected to our public eth2 testnet

File: README.md [hunk 7]
```diff
@@ -1,28 +1,36 @@
...
+A more detailed set of installation and usage instructions as well as breakdowns of each individual component are available in the [official documentation portal](https://prysmaticlabs.gitbook.io/prysm/). If you still have questions, feel free to stop by either our [Discord](https://discord.gg/KSA7rPr) or [Gitter](https://gitter.im/prysmaticlabs/geth-sharding?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) and a member of the team or our community will be happy to assist you.
 
 **Interested in what's next?** Be sure to read our [Roadmap Reference Implementation](https://github.com/prysmaticlabs/prysm/blob/master/docs/ROADMAP.md) document. This page outlines the basics of sharding as well as the various short-term milestones that we hope to achieve over the coming year.
 
 ### Come join the testnet!
```
Comment (rauljordan) [inline README.md]: See comment below about moving this to the section on `Connecting to the Public Testnet`

File: README.md [hunk 8]
```diff
@@ -187,20 +197,59 @@ bazel run //validator
...
 
+## Setting up a local ETH2 development chain
+
+This section outlines the process of setting up Prysm for local interop testing with other Ethereum 2.0 client implementations. See the [INTEROP.md](https://github.com/prysmaticlabs/prysm/blob/master/INTEROP.md) file for advanced configuration options. For more background information on interoperability development, see [this blog post](https://blog.ethereum.org/2019/09/19/eth2-interop-in-review/).
+
+### Installation and Dependencies
+
+To begin setting up a local ETH2 development chain, follow the **Bazel** instructions found in the [dependencies](#dependencies) and [installation](#installation) sections respectively. After completion, continue on to [generating a genesis state](#generating-a-genesis-state).
+
+### Genesis parameters
```
Comment (rauljordan) [inline README.md]: Let's get rid of this from line 208-213, we're not generating a genesis state for these instructions, we're just running a local beacon chain + validator

File: README.md [hunk 9]
```diff
@@ -187,20 +197,59 @@ bazel run //validator
...
+
+- **--genesis-time** uint: Unix timestamp used as the genesis time in the generated genesis state (defaults to now)
+- **--mainnet-config** bool: Select whether genesis state should be generated with mainnet or minimal (default) params
+- **--num-validators** int: Number of validators to deterministically include in the generated genesis state
+- **--output-ssz** string: Output filename of the SSZ marshaling of the generated genesis state
+
+### Generating a genesis state
```
Comment (rauljordan) [inline README.md]:
```suggestion
### Running the beacon node + validator with Bazel
```

**Pull request merged**